base/applications/network/net/lang/en-US.rc

```diff
@@ -0,0 +1,8 @@
+#include "resource.h"
```

Comment:
I don't remember we need to include header files in RC files :thinking:

Reply:
They are only in the main .rc file (the one that includes these per-language "sub"-rc files)

Reply:
@HBelusca okey, I 'll take it into account.